docs/templates/reachview/firmware-reflashing.mdx

```diff
@@ -42,7 +42,7 @@ We recommend using the terminal to install the Flash Tool on your Linux PC. Foll
 
 * Press `Ctrl+Alt+T` to open the terminal
 * Navigate to the directory with the downloaded file with `cd ~/Downloads` command
-* Run `sudo dpkg -i reach-firmware-flash-tool_1.2.0_amd64.deb` to start installation
+* Run `sudo dpkg -i reach-firmware-flash-tool_1.3.0_amd64.deb` to start installation
 
 !!! danger ""
 	Please confirm adding the current user to `plugdev`, `dialout` and `lpadmin` groups during the installation process. In case of a negative answer, the application might not work as expected.
```

Comment:
Please remove `lpadmin` group from the list. We use only `plugdev` and `dialout` now.